### ISSUE TITLE
I transferred coin to my linux wallet, and the transaction was successful. Why is my balance on my node query 0?

### ISSUE THREAD

**Issue by tzdadada** [turtlecoin/turtlecoin#1002]
I transferred coin to my linux wallet, and the transaction was successful. Why is my balance on my node query 0?

Transaction information: https://explorer.turtlecoin.lol/transaction.html?hash=9a264e44750ad77b3962da3ea6a46a71e98b32239ca94e470c5dc0afa175d384

Returns the result:
{
     "locked": 0,
     "unlocked": 0
}

Am I missing any steps?

**Comment by brandonlehmann** [comment]
Is your node (TurtleCoind) fully synced? Is your wallet fully sycned? Does the address in your wallet match the one you sent the funds to?

**Comment by tzdadada** [comment]
@brandonlehmann 
I have determined that my node height is the latest height, and also compared the address, it is indeed the address of my wallet ："TRTLv36F7qrWswPDfCDeuPQBCR8ZePERRVvTkxGE1T7c7NMpzCf29h722gZvSu8JhwhxtTWwgiWYoSTmr69y4898YBESS5ZAWaV"

My operation flow is: use the "zedwallet" command to create the first wallet, then  `curl -X POST "http://127.0.0.1:8448/wallet/open" -H "accept: application/json" -H "X-API-KEY: password" -H "Content-Type: application/json" -d "{ \"daemonHost\": \"0.0.0.0\", \"daemonPort\": 8448, \"filename\": \"trtl.wallet\", \"password\": \"password\"}"  `, open the wallet, and then generate the address, but there is still no balance in the transfer to the address.

What steps have I missed?

**Comment by tzdadada** [comment]
When replying to you, my height is:  Beijing time： `2020-Jan-15 14:22:32.355051 INFO    New Top Block Detected: 2194737`

**Comment by tzdadada** [comment]
>  `Is your wallet fully sycned? `
I should be missing this step. How do I get the wallet fully synchronized?

**Comment by tzdadada** [comment]
My startup commands are :

` ./TurtleCoind --rpc-bind-ip=0.0.0.0 --rpc-bind-port=1111`

 `./wallet-api --rpc-bind-ip 0.0.0.0 --port 2222 --rpc-password "password" --enable-cors "*" --scan-coinbase-transactions`

**Comment by brandonlehmann** [comment]
> @brandonlehmann
> I have determined that my node height is the latest height, and also compared the address, it is indeed the address of my wallet ："TRTLv36F7qrWswPDfCDeuPQBCR8ZePERRVvTkxGE1T7c7NMpzCf29h722gZvSu8JhwhxtTWwgiWYoSTmr69y4898YBESS5ZAWaV"
> 
> My operation flow is: use the "zedwallet" command to create the first wallet, then `curl -X POST "http://127.0.0.1:8448/wallet/open" -H "accept: application/json" -H "X-API-KEY: password" -H "Content-Type: application/json" -d "{ \"daemonHost\": \"0.0.0.0\", \"daemonPort\": 8448, \"filename\": \"trtl.wallet\", \"password\": \"password\"}" `, open the wallet, and then generate the address, but there is still no balance in the transfer to the address.
> 
> What steps have I missed?

So you created a new wallet, opened the new wallet, and the balance is 0? That would be correct unless you send it funds.

**Comment by tzdadada** [comment]
@brandonlehmann 
Yes, I have already sent money to it, but the balance is still 0.

**Comment by brandonlehmann** [comment]
When did you send funds to it? What is the transaction hash?

**Comment by tzdadada** [comment]
@brandonlehmann 
A transaction was made yesterday and a few days ago, and the transaction hash:783ab259b4aa80aba2cb9cc9b1a7201143bbaccd56d146a446844e8b17d28e1b

 9a264e44750ad77b3962da3ea6a46a71e98b32239ca94e470c5dc0afa175d384


Please, check it out for me,thanks.

**Comment by tzdadada** [comment]
@brandonlehmann 
I restarted "TurtleCoind" and then synced it. I found the remittance I lost before. I guess the problem is on my node.

`New transaction found!

Incoming transfer:
Hash: 9a264e44750ad77b3962da3ea6a46a71e98b32239ca94e470c5dc0afa175d384
Block height: 2188348
Timestamp: 2020-01-13 08:51
Amount: 130,000.00 TRTL

2189923 of 2196172
2190623 of 2196172
2191323 of 2196172
2192123 of 2196172
2192923 of 2196195

New transaction found!

Incoming transfer:
Hash: 783ab259b4aa80aba2cb9cc9b1a7201143bbaccd56d146a446844e8b17d28e1b
Block height: 2192363
Timestamp: 2020-01-14 18:28
Amount: 108,000.00 TRTL`


` 1      advanced                 List available advanced commands
 2      address                  Display your payment address
 3      balance                  Display how much TRTL you have
 4      backup                   Backup your private keys and/or seed
 5      exit                     Exit and save your wallet
 6      help                     List this help message
 7      transfer                 Send TRTL to someone

[TRTL trtl]: 3
Available balance: 238,000.00 TRTL
Locked (unconfirmed) balance: 0.00 TRTL
Total balance: 238,000.00 TRTL`

**Comment by tzdadada** [comment]
```
New transaction found!

Incoming transfer:
Hash: 9a264e44750ad77b3962da3ea6a46a71e98b32239ca94e470c5dc0afa175d384
Block height: 2188348
Timestamp: 2020-01-13 08:51
Amount: 130,000.00 TRTL

2189923 of 2196172
2190623 of 2196172
2191323 of 2196172
2192123 of 2196172
2192923 of 2196195

New transaction found!

Incoming transfer:
Hash: 783ab259b4aa80aba2cb9cc9b1a7201143bbaccd56d146a446844e8b17d28e1b
Block height: 2192363
Timestamp: 2020-01-14 18:28
Amount: 108,000.00 TRTL
```
```
1 advanced List available advanced commands
2 address Display your payment address
3 balance Display how much TRTL you have
4 backup Backup your private keys and/or seed
5 exit Exit and save your wallet
6 help List this help message
7 transfer Send TRTL to someone

Available balance: 238,000.00 TRTL
Locked (unconfirmed) balance: 0.00 TRTL
Total balance: 238,000.00 TRTL
```

**Comment by brandonlehmann** [comment]
Awesome.
